mcs/class/corlib/System/RuntimeTypeHandle.cs

```diff
@@ -258,5 +258,7 @@ internal static bool IsSubclassOf (RuntimeType childType, RuntimeType baseType)
 		[MethodImplAttribute(MethodImplOptions.InternalCall)]
 		internal extern static bool is_subclass_of (IntPtr childType, IntPtr baseType);
 
+		[MethodImplAttribute(MethodImplOptions.InternalCall)]
```

Comment:
Please add `[PreserveDependencyAttribute (".ctor()", "System.Runtime.CompilerServices. IsByRefLikeAttribute"` for linker tracking